lib/ruby-mpd/parser.rb

```diff
@@ -150,17 +160,39 @@ def parse_response(command, string)
       build_response(command, string)
     end
 
+    def parse_command_list(commands, string)
+      [].tap do |results|
+        string.split("list_OK\n").each do |str|
```

Comment:
you can get rid of `tap` by using `string.split("list_OK\n").each_with_object([]) do |str, results|`